VCCrypto/VCCrypto/Algo/Secp256k1.swift

```diff
@@ -18,19 +18,37 @@ enum Secp256k1Error: Error {
 
 public struct Secp256k1: Signing {
     
-    public init() {}
+    private let publicKey: Secp256k1PublicKey?
+    
+    private let secret: VCCryptoSecret?
+    
+    private let hashOperation = Sha256()
+    
+    /// Create Secp256k1 object from Secret, for signing.
+    public init(secret: VCCryptoSecret) throws {
```

Comment:
Curious, why pull these to the constructor?

Reply:
reverted back to old implementation per discussion